api/go.mod

```
@@ -0,0 +1,12 @@
+module github.com/containerd/containerd/api
+
+go 1.16
+
+require (
+	github.com/containerd/ttrpc v1.0.2
+	github.com/containerd/typeurl v1.0.2
+	github.com/gogo/googleapis v1.4.1
+	github.com/gogo/protobuf v1.3.2
+	github.com/opencontainers/go-digest v1.0.0
+	google.golang.org/grpc v1.38.0
+)
```

Do we need `replace` rules here to match the main `go.mod` ? Looks like at least `googleapis` and `genproto` are missing;

```suggestion
)

replace (
	github.com/gogo/googleapis => github.com/gogo/googleapis v1.3.2
	google.golang.org/genproto => google.golang.org/genproto v0.0.0-20200224152610-e50cd9704f63
)
```

In that case, also update the comment in the main `go.mod`; https://github.com/containerd/containerd/blob/ee27cde735e2a27a5e1465116ac98593b78d0a6f/go.mod#L71

---

doing both suggestions.